lms_latest.json

```
@@ -0,0 +1,54 @@
+{
+	"Logitech Media Server - Latest Stable": {
+		"description": "Logitech Media Server (aka Squeezebox, slimserver, etc.). Latest stable version. For future updates see more information icon after first installation<p>Uses custom docker image: <a href='https://hub.docker.com/r/doliana/logitech-media-server' target='_blank'>https://hub.docker.com/r/doliana/logitech-media-server</a>. Available for amd64, arm/v7 &arm64 architecture.",
+		"more_info": "<h4>Update to newer version</h4><p>After first installation: to update to a newer stable version, either uninstall/reinstall the Rockon or use the Watchtower Rockon to manage the update of the underlying Media Server image, as it does not automatically update itself during a stop/restart of the Rockon like some other Rockons do. Don't use the update option within the LMS!</p><p>The LMS configuration files will remain in place on the volumes that were specified during the initial installation</p>",
+		"website": "https://mysqueezebox.com/",
+		"version": "2.2",
+		"ui": {
+			"slug": ""
+		},
+		"containers": {
+			"lms_latest": {
+				"image": "doliana/logitech-media-server",
+				"tag": "latest",
+				"launch_order": 1,
+				"opts": [
+					[
+						"--net",
+						"host"
+					]
+				],
+				"ports": {
+					"9000": {
+						"description": "WebUI port. Suggested default: 9000",
```

As with the previous LMS Rock-on, we should include that it MUST be set to 9000.
```suggestion
						"description": "WebUI port. This MUST be set to 9000",
```